spec/ttml1.xml

```diff
@@ -8581,6 +8581,13 @@ agent with another agent that portrays the character.</p>
 <p>The <att>agent</att> attribute of a <el>ttm:actor</el> element must
 reference a <emph>significant</emph> <el>ttm:agent</el> element that denotes the agent
 acting the part of a character.</p>
+<note role="clarification">
+<p>The semantics associated with an <el>ttm:actor</el> element referencing its parent 
+<el>ttm:agent</el> element are not defined. In the case of an actor playing themselves, 
+like Steve Coogan in The Trip playing Steve Coogan, two <el>ttm:agent</el> elements are used instead: 
+the first element (corresponding to Steve Coogan the person) has <att>type</att> attribute equal to <code>person</code>, 
+while the second element (corresponding to Steve Coogan the character) has (i) a <att>type</att> attribute equal to <code>character</code> and (ii) a child <el>ttm:actor</el> element that references the first <el>ttm:agent</el> element.</p>
```

Comment:
This wording is fine but doesn't address the need to warn implementers that special handling is required. Perhaps something like:

_Despite the lack of defined semantics, it is nevertheless syntactically permitted for a `ttm:actor` element to reference its parent `ttm:agent` element; implementers should take care to handle this situation, for example avoiding infinite loops etc._